apps/rig_inbound_gateway/lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -34,6 +35,13 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
       build_url(api["proxy"], request_path)
       |> possibly_add_correlation_id(response_from)
 
+    host_ip = Confex.fetch_env!(:rig_inbound_gateway, RigInboundGatewayWeb.Endpoint)[:url][:host]
+
+    req_headers =
+      req_headers
+      |> HttpHeader.put_host_header(url)
```

Comment:
`url` contains the URL as built by `build_url/2`. What is wrong with `url`?

Reply:
As you were referring explicitily in the readme update, that  _"`target_url` defined by the proxy configuration"_ should be passed to the host header, so I thought this could somehow be reflected in the variable name.
Your choice!

Reply:
ah, now I understand. Well, the name `target_url` - for better or worse - refers to the host only and doesn't include the request path. `url`, on the other hand, does include the path and shouldn't be confused with the `target_url` field as defined in the API configuration. So your intuition is good, but I guess we should stick to the names we've got, at least until we get to revise the API configuration file for using more descriptive names.

Reply:
kk - thx for explaination